nullaway/src/main/java/com/uber/nullaway/handlers/Handler.java

```diff
@@ -275,9 +276,21 @@ void onDataflowVisitLambdaResultExpression(
    *
    * @param expr The AST node for the expression being matched.
    * @param state The current visitor state.
-   * @return If the method call in the expression is to Optional.get().
+   * @param errorMessage error message string and type of the error wrapped in {@link
+   *     com.uber.nullaway.NullAway.ErrorMessage}.
    */
-  boolean checkIfOptionalGetCall(ExpressionTree expr, VisitorState state);
+  void getErrorMessage(ExpressionTree expr, VisitorState state, NullAway.ErrorMessage errorMessage);
+
+  /**
+   * Called when the store access paths are filtered for local variable information before an
+   * expression.
+   *
+   * @param accessPath The access path that needs to be checked if filtered.
+   * @param state The current visitor state.
+   * @return true if the accesspath should be filtered to be included in the local variable
```

Comment:
"should be filtered to be included" doesn't easily parse in my mind. Are we excluding or including? (e.g. is this a whitelist or a blacklist?).

Reply:
After reading the rest of the code more carefully, I'd go with "@return true if the nullability information for this accesspath should be treated as part of the surrounding context when processing a lambda expression or anonymous class declaration"

I'd also rename  `getLocalVarInfoBefore` to e.g. `getNullnessInfoBeforeNewContext`, to more accurately reflect the fact that it preserves info for APs other than locals. But let's see if that makes sense to @msridhar as well before going ahead with that renaming for a core API method :)

If we do rename that, we can have this method be `Handler.includeApInfoInSavedContext`, or something like that.

Reply:
I agree with @lazaroclapp's comments and suggested names 😄

Reply:
👍Done